src/services/smart-contract.service.ts

```diff
@@ -24,6 +25,30 @@ async function setAppConfig(appName: string, socialAuth: AppConfigCred[]) {
   await signTransaction('setAppConfig', [appConfig])
 }
 
+async function enableUiMode() {
+  await signTransaction('setUiMode')
+}
+
+async function grantDelegate(
+  keyAddress: string,
+  permissions: DelegatePermission[],
+  appId: AppId
```

Comment:
make this (appId, keyAddress, permissions) or turn this into object

Reply:
https://github.com/arcana-network/developer-dashboard/pull/156/commits/7b6e188f7d1078eddc2f39cf7bc45158ec35f195